tests/test_player_with_strategy_one.py

```diff
@@ -9,18 +8,50 @@ def setUp(self):
         self.player_blue = PlayerWithStrategyOne('blue')
         self.player_red = PlayerWithStrategyOne('red')
 
-    def test_player_wins_with_strategy_one(self):
-        self.board_player_blue_wins = GameBoard.from_matrix([
-                    ['blue', 'blue', 'blue',  'red',   None,   None, None],
-                    ['red',   'red',  'red', 'blue', 'blue',   None, None], # <- blue row wins
-                    ['blue',  'red', 'blue',  'red', 'blue',   None, None],
-                    ['blue',  'red', 'blue',  'red',  'red',  'red', None],
-                    ['blue', 'blue',  'red', 'blue',  'red', 'blue', None],
-                    ['red',   'red',  'red', 'blue',  'red', 'blue', None]])
-        self.match = Match(self.player_blue, self.player_red, self.board_player_blue_wins)
-        self.assertFalse(self.player_blue.is_winner(self.board_player_blue_wins))
-        for i in range(8):
-            self.match.play_next_turn()
-            self.assertFalse(self.player_blue.is_winner(self.board_player_blue_wins))
-        self.match.play_next_turn()
-        self.assertTrue(self.player_blue.is_winner(self.board_player_blue_wins))
+    def test_player_with_strategy_one_blue_wins_on_right_upper_corner(self):
```

Comment:
Even though the names are ok, a comment explaining what are you testing would be really nice (note: explain the case you are testing, not how you are testing it).